### PR TITLE
[bug 1183422] Add waffle switch for new AAQ

### PR DESCRIPTION
**Pull request by rehandalal**
r?

### CODE REVIEW

File: kitsune/questions/migrations/0004_new_aaq_waffle_flag.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('waffle', '0001_initial'),
+        ('questions', '0003_auto_20150430_1304'),
+    ]
+
+    operations = [
+        migrations.RunPython(create_waffle_flag),
```
Comment (mythmon) [inline kitsune/questions/migrations/0004_new_aaq_waffle_flag.py]: This migration is easily reversible, by simply deleting the waffle flag. You can do that by passing a second function to `RunPython` for the reverse direction.

**Comment by mythmon** [comment]
I tried this out locally. Both flows work nicely.

The failing test appears to be my fault. That didn't happen on my AAQ auth PR because, for some reason, Travis didn't run on my commit. I think we can just delete the test in question, and probably the middleware too. I can do that, if you prefer.

**Comment by rehandalal** [comment]
i'll fix the test and add the delete method

**Comment by rehandalal** [comment]
updated ^

**Comment by mythmon** [comment]
r+ when we get :tea:.

**Pull request merged**